config/community-resources.json

```
@@ -98,5 +98,15 @@
       "name": "Maximising Your Job Prospects in Tech with BioDrop",
       "url": "https://loftwah.medium.com/maximising-your-job-prospects-in-tech-with-BioDrop-2fea408301cc"
     }
+  },
+  {
+    "name": "Nafisa Faruna",
+    "username": "Nenepo",
```

This should be the username of the resource author 

```suggestion
    "username": "KimMcMahon",
```